detectron2/export/__init__.py

```diff
@@ -9,7 +14,13 @@
 else:
     from .api import *
 
-from .flatten import TracingAdapter
-from .torchscript import scripting_with_instances, dump_torchscript_IR
+
+STABLE_ONNX_OPSET_VERSION = 11
```

Comment:
does it make sense to condition this value based on `torch.__version__`? e.g. set to 16 for newer torch versions

Reply:
Not really, pytorch and ONNX opset versions is a 1:N relationship. Many model authors experiment performance using different combinations of opset and/or pytorch versions. Many time newer pytorch versions improve implementation of older opsets... Other times, newer opsets have new operators available, which allows more efficient implementation

It is up to the authors pick their preferred (possibly older) opset version on, generally speaking, the latest pytorch version